### ISSUE TITLE
Support correct save of numbers in admin template

### ISSUE THREAD

**Issue by DutchmanNL** [ioBroker/create-adapter#351]
Currently using numbers in the admin will store its content as string, this can be challenging is station you need a number for calculation purposes.

this part does only provide a string to the variable : https://github.com/ioBroker/create-adapter/blob/master/templates/admin/index_m.html.ts#L80-L92

Quick workaround I made for now is : 

```
		function save(callback) {
			// example: select elements with class=value and build settings object
			var obj = {};
			$('.value').each(function () {
				var $this = $(this);
				if ($this.attr('type') === 'checkbox') {
					obj[$this.attr('id')] = $this.prop('checked');
				} else if ($this.attr('type') === 'number'){
					obj[$this.attr('id')] = parseInt($this.val());
				} else {
					obj[$this.attr('id')] = $this.val();
				}
			});
			callback(obj);
		}
```

this is only a quick and dirty fix as number with "comma" are not correctly converted by this code

**Comment by AlCalzone** [comment]
Should we maybe use parseFloat instead?

**Comment by Apollon77** [comment]
Check if step property is set and if to a not full number ... then assume floats are allowed and then use parseFloat?

**Comment by AlCalzone** [comment]
`parseFloat("1") === 1`, so why so complicated?

**Comment by Apollon77** [comment]
because also validation matters or? We should be able to define if int or float is wanted by a specific field

**Comment by AlCalzone** [comment]
Don't those fields have browser integrated validation?

**Comment by Apollon77** [comment]
No idea :-)

**Comment by DutchmanNL** [comment]
> Don't those fields have browser integrated validation?

yes they have, but that does not fix my problem !
The brower validates if the input is a number by type=number, great. :)

but the variable typ in iobroker is a string, here we have a problem.
Mostly this doesn't matter, as we use connection times and such kind of things and multiply them javascript is smart enough to convert it to a number.

BUT in my case I have a config item "max_count" and must check if a current value <== this max_count or === max_count
As this max count is a string and not a number, I can't do that...

Okey I could do <= or == instead but that's not "clean programming"

**Comment by AlCalzone** [comment]
@DutchmanNL The original problem is clear and we have to solve it. But...
We're discussing how to best convert the value from the text field into a number. Basically there are two options: `parseInt`, which always produces whole numbers and `parseFloat` which produces floats or integers (depending on the input). 
The question is: can we always use `parseFloat` or are there maybe problems with decimal separators (dot vs comma)?

**Comment by AlCalzone** [comment]
fixed in #634
